kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -326,6 +348,34 @@ createPatchJsonToUpdateClusterPolicy() {
   eval $__result="'${patchJsonVal}'"
 }
 
+#
+# Function to create patch json to update cluster server start policy
+# Added on 08/16/2022
+# $1 - Cluster resource in json format
+# $2 - Name of cluster whose policy will be patched
+# $3 - policy value of "IfNeeded" or "Never"
+# $4 - Return value containing patch json string
+#
+createPatchJsonToUpdateClusterPolicyUsingClusterResource() {
+  local clusterJson=$1
+  local clusterName=$2
+  local policy=$3
+  local __result=$4
+  local addClusterStartPolicyCmd=""
+  local mapCmd=""
+  local existingClusters=""
+  local patchJsonVal=""
+  local startPolicyPatch=""
+
+  existingClusters=$(echo ${clusterJson} | jq -cr '(.spec.clusters)')
```

Comment:
My understanding is that cluster resource is specific to a particular cluster and unlike domain resource there's no `clusters` attribute under the `spec` in the cluster resource (instead it has a `clusterName` attribute).

Reply:
Changed